pagerduty_connector.py

```diff
@@ -119,22 +118,29 @@ def _parse_response(self, result, r):
             try:
                 resp_data = r.json()
             except:
-                result.set_status(phantom.APP_ERROR, "Unable to parse response as a JSON status_code: {0}, data: {1}".format(r.status_code, self._normalize_text(r.text)))
+                result.set_status(phantom.APP_ERROR, "Unable to parse response as a JSON status_code: {0}, data: {1}"
+                .format(r.status_code, self._normalize_text(r.text)))
```

Comment:
I'm surprised the linter is fine with this 🤔 usually there is one or more indent levels for continuation lines to distinguish them from the actual start of a line

Reply:
could we update all of these so there's some level of indentation if it's a continuation line?